### PR TITLE
Add type hint for OC_Channel

### PR DESCRIPTION
**Pull request by VicDeo**
Ref https://github.com/owncloud/core/issues/16574

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by VicDeo** [comment]
@DeepDiver1975 is it better now?

File: lib/private/util.php [hunk 1]
```diff
@@ -393,6 +393,7 @@ private static function loadVersion() {
 			if (\OC::$server->getSystemConfig()->getValue('installed', false)) {
 				$channel = \OC::$server->getAppConfig()->getValue('core', 'OC_Channel');
 			} else {
+				/** @var $OC_Channel string */
```
Comment (rullzer) [inline lib/private/util.php]: You should define the type of OC_Channel where OC_Channel is defined ;)
So in version.php

**Comment by rullzer** [comment]
So I was looking at the code around it https://github.com/owncloud/core/blob/master/lib/private/util.php#L391-L404

That part already looks weird... $channel is always defined...

**Comment by VicDeo** [comment]
@rullzer FYI I'm 'fixing' https://github.com/owncloud/core/issues/16574 here.
while I don't think this is a bug really.

**Comment by MorrisJobke** [comment]
Yes. This fixes the issue in PHPStorm and it is also solved in that way for the other variables that are imported via include: https://github.com/owncloud/core/pull/16592/files#diff-dd3121bfbeff46c08c30cb0666aa701eR388

:+1:

**Pull request merged**